src/ucs/sys/rcache.c

```
@@ -516,7 +516,8 @@ ucs_rcache_create_region(ucs_rcache_t *rcache, void *address, size_t length,
     region->refcount = 1;
     region->status = status =
         UCS_PROFILE_NAMED_CALL("mem_reg", rcache->params.ops->mem_reg,
-                               rcache->params.context, rcache, arg, region);
+                               rcache->params.context, rcache, arg, region,
+                               merged ? 0 : UCS_RCACHE_MEM_REG_HIDE_ERRORS);
```

need to hide errors if merged, not vice-versa